crates/swc_core/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
-authors = ["강동윤 <kdy1997.dev@gmail.com>", "OJ Kwon <kwon.ohjoong@gmail.com>"]
+authors     = ["강동윤 <kdy1997.dev@gmail.com>", "OJ Kwon <kwon.ohjoong@gmail.com>"]
 description = "TBD"
-edition = "2021"
-license = "Apache-2.0"
-name = "swc_core"
-repository = "https://github.com/swc-project/swc.git"
-version = "0.2.0"
+edition     = "2021"
+license     = "Apache-2.0"
+name        = "swc_core"
+repository  = "https://github.com/swc-project/swc.git"
+version     = "0.2.0"
 
-[package.metadata.docs.rs]
-all-features = true
-rustdoc-args = ["--cfg", "docsrs"]
+  [package.metadata.docs.rs]
+  all-features = true
```

Comment:
We may need to modify this.
https://docs.rs/crate/swc_core/0.2.0/builds

Reply:
Not sure what'd be good way to fix - tried to limit number of features for now.